package.json

```diff
@@ -147,13 +147,13 @@
     "cypress-cucumber-preprocessor": "1.12.0",
     "cypress-pipe": "1.3.3",
     "del-cli": "1.1.0",
-    "electron": "4.2.4",
-    "electron-builder": "20.38.3",
+    "electron": "7.0.0",
+    "electron-builder": "21.2.0",
```

Comment:
Please don't update the version of all the electron dependencies. @osvaldovega works on another issue that will take care of all of those upgrades.